pwndbg/commands/context.py

```diff
@@ -162,11 +163,26 @@ def context_disasm():
 
     return banner + result
 
+# XXX: move this part to color/ just like color/disasm.py:instrution()?
 theme.Parameter('highlight-source', True, 'whether to highlight the closest source line')
-
+source_code_length = pwndbg.config.Parameter('context-source-code-length',
+                                             10,
+                                             'number of lines of source code to printed by the context command')
```

Comment:
Maybe: `number of source code lines to print by the context command`.

Reply:
Fixed